src/main/java/io/github/communitymod/capabilities/entitylevel/MobLevelEventHandler.java

```diff
@@ -0,0 +1,246 @@
+            return 42;
+        } else if (rnd <= Math.pow(2, 30.5)) {
+            return 41;
+        } else if (rnd <= Math.pow(2, 31)) {
+            return 40;
+        } else if (rnd <= Math.pow(2, 31.5)) {
+            return 39;
+        } else if (rnd <= Math.pow(2, 32)) {
+            return 38;
+        } else if (rnd <= Math.pow(2, 32.5)) {
+            return 37;
+        } else if (rnd <= Math.pow(2, 33)) {
+            return 36;
+        } else if (rnd <= Math.pow(2, 33.5)) {
+            return 35;
+        } else if (rnd <= Math.pow(2, 34)) {
+            return 34;
+        } else if (rnd <= Math.pow(2, 34.5)) {
+            return 33;
+        } else if (rnd <= Math.pow(2, 35)) {
+            return 32;
+        } else if (rnd <= Math.pow(2, 35.5)) {
+            return 31;
+        } else if (rnd <= Math.pow(2, 36)) {
+            return 30;
+        } else if (rnd <= Math.pow(2, 36.5)) {
+            return 29;
+        } else if (rnd <= Math.pow(2, 37)) {
+            return 28;
+        } else if (rnd <= Math.pow(2, 37.5)) {
+            return 27;
+        } else if (rnd <= Math.pow(2, 38)) {
+            return 26;
+        } else if (rnd <= Math.pow(2, 38.5)) {
+            return 25;
+        } else if (rnd <= Math.pow(2, 39)) {
+            return 24;
+        } else if (rnd <= Math.pow(2, 39.5)) {
+            return 23;
+        } else if (rnd <= Math.pow(2, 40)) {
+            return 22;
+        } else if (rnd <= Math.pow(2, 40.5)) {
+            return 21;
+        } else if (rnd <= Math.pow(2, 41)) {
+            return 20;
+        } else if (rnd <= Math.pow(2, 41.5)) {
+            return 19;
+        } else if (rnd <= Math.pow(2, 42)) {
+            return 18;
+        } else if (rnd <= Math.pow(2, 42.5)) {
+            return 17;
+        } else if (rnd <= Math.pow(2, 43)) {
+            return 16;
+        } else if (rnd <= Math.pow(2, 43.5)) {
+            return 15;
+        } else if (rnd <= Math.pow(2, 44)) {
+            return 14;
+        } else if (rnd <= Math.pow(2, 44.5)) {
+            return 13;
+        } else if (rnd <= Math.pow(2, 45)) {
+            return 12;
+        } else if (rnd <= Math.pow(2, 45.5)) {
+            return 11;
+        } else if (rnd <= Math.pow(2, 46)) {
+            return 10;
+        } else if (rnd <= Math.pow(2, 46.5)) {
+            return 9;
+        } else if (rnd <= Math.pow(2, 47)) {
+            return 8;
+        } else if (rnd <= Math.pow(2, 47.5)) {
+            return 7;
+        } else if (rnd <= Math.pow(2, 48)) {
+            return 6;
+        } else if (rnd <= Math.pow(2, 48.5)) {
+            return 5;
+        } else if (rnd <= Math.pow(2, 49)) {
+            return 4;
+        } else if (rnd <= Math.pow(2, 49.5)) {
+            return 3;
+        } else if (rnd <= Math.pow(2, 50)) {
+            return 2;
+        } else if (rnd <= Math.pow(2, 50.5)) {
+            return 1;
+        } else {
+            return 1;
+        }
```

Comment:
True

Reply:
Ok I think I found a probable solution:

```java
Double lambda = 0.5; // Tweak this value for the desired distribution
return Math.min(Math.round(Math.log(1-Math.random())/-lambda), 100);
```
 Or, more readably,
```java
Double lambda = 0.5; // Tweak this value for the desired distribution
Double randomNumber = Math.random();
int calculated = Math.round(Math.log(1-randomNumber)/-lambda);
return Math.min(calculated, 100); // Cap it off at 100 in case it went over
```

Reply:
Fixed, thanks MetalTurtle18!